ethcontract/src/transaction/build.rs

```diff
@@ -36,7 +35,7 @@ impl<T: Transport> TransactionBuilder<T> {
                 build_transaction_request_for_local_signing(
                     self.web3,
                     None,
-                    gas_price,
+                    self.gas_price,
```

Comment:
We might even be able to move this to the `TransactionOptions` as a field now.

Reply:
True :) Moved.